build.gradle

```diff
@@ -135,12 +135,12 @@ bintray {
 ext {
     lombokVersion = '1.18.8'
     springCloudVersion = 'Hoxton.SR3'
-    feignVersion = '10.2.3'
-    feignFormVersion = '3.5.0'
-    wiremockVersion = '2.24.1'
+    feignVersion = '10.9'
+    feignFormVersion = '3.8.0'
+    wiremockVersion = '2.26.3'
 }
 
-ext["spring-cloud-openfeign.version"] = "2.1.2.RELEASE"
+ext["spring-cloud-openfeign.version"] = "2.2.2.RELEASE"
```

Comment:
how about reviewing existing PR first? #65

Reply:
I need to use new form version, as consequence of it I changed them

Reply:
aye, just noticed there's a new annotation. fixing #65 as well. with feign 10.9 form also comes in as 3.8 - there is no need to include it separately in dependency set. check out that pr - should be all clear and help out this one as well